cmd/qat_plugin/README.md

```diff
@@ -245,10 +258,190 @@ You can verify this on your nodes by checking for the relevant PCI identifiers:
 for i in 0442 0443 18a1 37c9 6f55 19e3 4941 4943; do lspci -d 8086:$i; done
 ```
 
+## NOTES
+1. The initcontainer deployment will split the qat resources into two: cy and dc.
+2. cy stands for crypto and dc for compression. Depending on what your workload needs, you can then request the either for the container.
+
+
 [1]:https://www-ssl.intel.com/content/www/us/en/design/products-and-solutions/processors-and-chipsets/purley/intel-xeon-scalable-processors.html
 [2]:https://www.intel.com/content/www/us/en/design/products-and-solutions/processors-and-chipsets/denverton/ns/atom-processor-c3000-series.html
 [3]:https://www.intel.com/content/www/us/en/ethernet-products/gigabit-server-adapters/quickassist-adapter-8950-brief.html
 [6]:https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md
 [7]:https://www.intel.com/content/www/us/en/content-details/710060/intel-quickassist-technology-software-for-linux-programmer-s-guide-hw-version-1-7.html
 [8]:https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md#build-openssl-intel-qat-engine-container
 [9]:https://www.intel.com/content/www/us/en/content-details/710059/intel-quickassist-technology-software-for-linux-getting-started-guide-hw-version-1-7.html
+
+
+# ERRORS and Troubleshooting:
+
+# Error-1  
+logs -n default           intel-qat-plugin-fjqnw
+(base) [root@spr-bkc-pc-3 ~]# k logs -n default           intel-qat-plugin-fjqnw
+
+Device scan failed: open /sys/bus/pci/drivers/vfio-pci/new_id: no such file or directory
+write to driver failed: 8086 4941
+github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv.writeToDriver
+        /go/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv/dpdkdrv.go:444
+github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv.(*DevicePlugin).setupDeviceIDs
+        /go/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv/dpdkdrv.go:195
+github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv.(*DevicePlugin).Scan
+        /go/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/qat_plugin/dpdkdrv/dpdkdrv.go:209
+github.com/intel/intel-device-plugins-for-kubernetes/pkg/deviceplugin.(*Manager).Run.func1
+        /go/src/github.com/intel/intel-device-plugins-for-kubernetes/pkg/deviceplugin/manager.go:100
+runtime.goexit
+        /usr/local/go/src/runtime/asm_amd64.s:1594
+failed to set device ID 4941 for vfio-pci. Driver module not loaded?
+
+## Solution-1
+modprobe vfio-pci
+
+## Error-2 - invalid options dpdk-test-crypto-perf
```

Comment:
I think we can drop this error and remove the manual testing command. It's a bit outdated anyways.